tests/conftest.py

```diff
@@ -49,13 +49,25 @@ def tls_ca_certificate_private_key_encrypted_path(tls_certificate_authority):
 
 
 @pytest.fixture
-def tls_certificate_pem_path(tls_certificate: trustme.LeafCert):
+def tls_certificate_private_key_path(tls_certificate: trustme.CA):
+    with tls_certificate.private_key_pem.tempfile() as private_key:
+        yield private_key
+
+
+@pytest.fixture
+def tls_certificate_key_and_chain_path(tls_certificate: trustme.LeafCert):
     with tls_certificate.private_key_and_cert_chain_pem.tempfile() as cert_pem:
         yield cert_pem
 
 
 @pytest.fixture
-def tls_ca_ssl_context(tls_certificate: trustme.LeafCert) -> ssl.SSLContext:
-    ssl_ctx = ssl.SSLContext()
-    tls_certificate.configure_cert(ssl_ctx)
+def tls_certificate_server_cert_path(tls_certificate: trustme.LeafCert):
+    with tls_certificate.cert_chain_pems[0].tempfile() as cert_pem:
+        yield cert_pem
```

Comment:
so here you're splitting what I called `tls_certificate_pem_path` in `tls_certificate_private_key_path` for just the private key and `tls_certificate_key_and_chain_path` for the bundle then add a test for just the bundle if I followed correctly ?

Reply:
sort of tls_certificate_pem_path was the wrong file - it was the CA key